wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/includes/searchpromotion_form.html

```diff
@@ -22,8 +22,15 @@
             </ul>
         {% endif %}
 
+        {% trans "Use an internal Page for this Promotion" %}
```

Comment:
```suggestion
        {% trans "Choose an internal page for this promotion" %}
```

OR

```suggestion
        {% trans "Select an internal page for this promotion" %}
```

No need for these to be proper nouns, let's replace 'use' with  'choose' or 'select'